src/PresentationalComponents/Common/__tests__/Tables.tests.js

```
@@ -87,3 +88,144 @@ describe('mapUpdateMethodFilterToAPISpec', () => {
+        values: [
+          { tagKey: 'group', value: 'prod' },
+          { tagKey: 'env', value: 'us-east-1' },
+        ],
+      },
+    ];
+
+    expect(buildTagFilter(tagFilters)).toEqual({
+      tags: [
+        'insights-client/group=prod',
+        'insights-client/env=us-east-1',
+      ],
```

ESLint

[error] Replace `⏎········'insights-client/group=prod',⏎········'insights-client/env=us-east-1',⏎······` with `'insights-client/group=prod',·'insights-client/env=us-east-1'`

(prettier/prettier)